### DISCUSSION TITLE
After clicking to enter the code file, the github page will flash

### DISCUSSION THREAD

**Discussion by KurumiRin** [community/community#57011] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

![Jun-02-2023 18-04-16](https://github.com/community/community/assets/52448921/8fac58e5-32d0-471f-b389-92a5d0dfb15e)

Just like the demonstration above, when entering src, the page will flicker.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @KurumiRin ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
